lms/djangoapps/teams/static/teams/js/collections/base.js

```diff
@@ -33,6 +35,8 @@
                     this.isStale = true;
                 },
 
+                /* TODO: These changes has been added to backbone.paginator
```

Comment:
Nit: I'd just use ``//`` for these lines. If you use ``/*`` then you should start a new line, like:

    /*
    TODO:: ...
    remove ...
    */